src/components/calcite-card/calcite-card.scss

```diff
@@ -0,0 +1,119 @@
+:host {
+  max-width: 100%;
+  & > a {
+    display: flex;
+    height: 100%;
+    flex-direction: column;
+    justify-content: space-between;
+    background-color: var(--calcite-ui-foreground);
+    transition: $transition;
+    position: relative;
+    border: 1px solid var(--calcite-ui-border-2);
+    color: var(--calcite-ui-text-3);
+    box-shadow: 0 0 0 rgba(0, 0, 0, 0);
+    &:hover {
```

Comment:
what about focus?

Reply:
Kind of related to the wrapping anchor being in the component vs. not - if it's not part of component we wouldn't be able to set consistent hover / active / focus styles